### PR TITLE
#8 create documentation

### LINKED ISSUE #15: 📝Architecture - Update diagram
Cc: @adamcogan @pierssinclairssw @christianmorfordwaitessw 

To Myself

### Tasks

- [x] Update the existing Architecture diagram

Thanks!

### LINKED ISSUE #16: 📝 Create the 8 important documents
Cc: @adamcogan @pierssinclairssw @christianmorfordwaitessw 

To Myself

Create documentation according to the [SSW Rule](https://www.ssw.com.au/rules/awesome-documentation)

### Tasks

- [x] Go through the important 8 documents in the rule
- [x] Create all documents

Thanks!

### PR DESCRIPTION
**Pull request by tkapa**
#16 #15 Creating the v1 of the 8 important documents

Changes:
- Adds all of the 8 important documents from the [SSW Rule](https://www.ssw.com.au/rules/awesome-documentation)

### CODE REVIEW

File: _docs/Instructions-Compile.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+
+### Install required tools 
+- Install nodejs from https://nodejs.org/en/ (>=v14.18.1 recommended)
+- Install yarn from https://classic.yarnpkg.com/en/ (>=v1.22.15 recommended)
+
+### Set up for development
+1. Clone the GitHub repository https://github.com/SSWConsulting/SSW.Website.git
+2. Run `yarn install` to install the required packages
+3. Create environment files (.env.development and .env.production) copying key values from `.env.template`
+    - Ask a member of the Website team (@pierssinclairssw, @tkapa) for the values of the environment keys
```
Comment (AttackOnMorty) [inline _docs/Instructions-Compile.md]: - [x] plz add me as Website team member lol

File: _docs/Instructions-Deployment.md [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+### Run Bicep Template
+
+Once all of the environment secrets are created, you are ready to reploy the required Azure Resources to your resource group.
+
+In your GitHub repository
+
+1. Navigate to **Actions | Staging - Deploy Azure Resources | Run workflow**
+2. Click **Run workflow**
+
+**Note: ** Running the `Production - Deploy Azure Resources` action will deploy the resources to your production environment.
```
Comment (AttackOnMorty) [inline _docs/Instructions-Deployment.md]: **Note: ** markdown does not take effect. 
- [x] Please remove the space after `:`

File: _docs/Instructions-Compile.md (hunk 1 above)
Comment (tkapa) [reply _docs/Instructions-Compile.md]: Added, I think I made this a week or so ago XD

**Review by tkapa — COMMENTED** [review]
Added Luke's suggested changes

**Review by AttackOnMorty — APPROVED** [review]
LGTM

**Pull request merged**